src/gui/gui.cpp

```diff
@@ -1186,6 +1186,7 @@ void FurnaceGUI::play(int row) {
   memset(chanOscBright,0,DIV_MAX_CHANS*sizeof(float));
   e->walkSong(loopOrder,loopRow,loopEnd);
   memset(lastIns,-1,sizeof(int)*DIV_MAX_CHANS);
+  if (wasFollowing=true) followPattern = true;
```

Comment:
This will set wasFollowing to true (`=` sets; `==` compares).
Remember to set wasFollowing to false afterwards.